ELSpace/Screens/Hub/Activity/ActivityController.swift

```diff
@@ -85,16 +81,10 @@ class ActivityController: ActivityControlling {
             .bind(to: isLoadingVar)
             .disposed(by: disposeBag)
 
-        Observable.of(
-            didFinishReportFetch.asObservable(),
-            didFinishProjectFetch.asObservable()
-        ).merge()
-            .map { [weak self] _ in
-                return self?.didFinishProjectFetch.value == true && self?.didFinishReportFetch.value == true
-            }.ignore(false)
-            .map { _ in () }
-            .bind(to: didFinishFetchSubject)
-            .disposed(by: disposeBag)
+        Observable.combineLatest(reportsSubject.asObservable().map { _ in () },
```

Comment:
Is it worth mapping elements (`map { _ in () }`) when you explicitly discard using them afterwords (`(_, _)`) ?